exercism/diamond/diamond.js

```diff
@@ -0,0 +1,44 @@
+var Diamond = function() {}
```

Comment:
Don't use var. 
let and const offer more than var does (scoping, clear distinction between immutability and mutability, etc).
Try to use ES6 where you can, just to practice.